staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -47,6 +49,11 @@ type proxyHandler struct {
 	proxyClientCert []byte
 	proxyClientKey  []byte
 
+	// Endpoints based routing to map from cluster IP to routable IP
+	enableAggregatorRouting bool
```

Comment:
https://github.com/kubernetes/kubernetes/pull/44742/files#r117992691 bubbles to here for an interface